src/sentry/conf/server.py

```diff
@@ -973,6 +973,11 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
         "schedule": crontab(minute=17),
         "options": {"expires": 3600},
     },
+    "span.descs.clusterer": {
+        "task": "sentry.ingest.span_clusterer.tasks.spawn_span_cluster_projects",
+        "schedule": crontab(minute=42),
```

Comment:
Since we're using the same queue as for transaction names, running this task with ~30 min difference to not overlap.